testing/functional/tests/cypress/fixtures/draftApplicationData.json

```diff
@@ -0,0 +1,22 @@
+{
+    "searchForYourSchool" : "Brentwood",
+
+    "schoolIWillBeAttending" : "College D 1st location",
+    "howWillYouAttendProgram" : "Part Time",
+    "programIWIllBeAttendingDropdownValue" : "Earth Science",
```

Comment:
this json property key is not a perfect camel case.

Reply:
Changes have been done.